plyer/facades/audio.py

```diff
@@ -41,7 +41,7 @@
 
 Supported Platforms
 -------------------
-Android
+Android, Windows, OS X
```

Comment:
```suggestion
Android, Windows, macOS
```

Reply:
Thanks for the review @misl6! I used `devicename`as reference but this totally works